sources/packages/api/src/services/notification/gc-notify.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from "@nestjs/common";
+import { ConfigService } from "..";
+import axios from "axios";
+import { GCNotifyResult, RequestPayload } from "./gc-notify.model";
+import { InjectLogger } from "../../common";
+import { LoggerService } from "../../logger/logger.service";
+
+@Injectable()
+export class GCNotifyService {
+  private readonly gcNotify;
```

Comment:
Just a suggestion, the variable name can be gcNotifyConfig to be more aligned with it's purpose.